cmd/protoc-gen-gorums/dev/ordering.go

```diff
@@ -13,12 +13,18 @@ import (
 	"google.golang.org/grpc/backoff"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	"google.golang.org/protobuf/types/known/anypb"
 )
 
+var numOrderedMethods int32 = 0
```

Comment:
This and the `getOrderedMethodID()` below is no longer needed.

Reply:
Oops, forgot about those